src/libraries/Microsoft.VisualBasic.Core/Directory.Build.props

```diff
@@ -1,9 +1,7 @@
 ﻿<Project>
   <Import Project="..\Directory.Build.props" />
   <PropertyGroup>
-    <MajorVersion>$([MSBuild]::Add($(MajorVersion), 6))</MajorVersion>
-    <PackageVersion>$(MajorVersion).$(MinorVersion).$(PatchVersion)</PackageVersion>
-    <AssemblyVersion>10.0.6.0</AssemblyVersion>
+    <AssemblyVersion>$([MSBuild]::Add($(MajorVersion), 6)).$(MinorVersion).0.0</AssemblyVersion>
```

Comment:
The previous AssemblyVersion was 10.x.x.x. This will now produce 12.x.x.x. What about 11.x.x.x? Intentionally left out? 😄

Reply:
just changed a second before ur comment

Reply:
Wondering if we should just increment the patch version instead so that we use the repo wide 11.x.x.x version in 6 years (lol).

Reply:
I don't think this is important at all^^